openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -579,12 +579,16 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
 
         # if there are multiple cameras, we need to add camera name
         if isinstance(col, (list, tuple)):
-            cam = [c for c in cameras if c in col[0]]
+            cam = next((c for c in cameras if c in col[0]), None)
         else:
             # in case of single frame
-            cam = [c for c in cameras if c in col]
+            cam = next((cam for cam in cameras if cam in col), None)
```

Comment:
```python
expected_filepath = col[0] is isinstance(col, (list, tuple)) else col
cam = next((cam for cam in cameras if cam in expected_filepath), None)
```
I feel like this would be clearer than the way it's written currently. Thoughts?